pageserver/src/page_service.rs

```diff
@@ -945,9 +945,8 @@ impl PageServerHandler {
             b"valid_until",
         )]))?
         .write_message_noflush(&BeMessage::DataRow(&[Some(
-            &valid_until.as_millis().to_be_bytes(),
-        )]))?
-        .write_message_noflush(&BeMessage::CommandComplete(b"SELECT 1"))?;
```

Comment:
@arpad-m do you remember why this CommandComplete was here originally?

Reply:
From my understanding, `SELECT <num_rows>` indicates the end of the message. I deleted this message because I saw another `CommandComplete` here, which should do the job?

https://github.com/neondatabase/neon/blob/3b2159310f7026686c29b27d2124bfceec58e8df/pageserver/src/page_service.rs#L1865

https://www.postgresql.org/docs/current/protocol-message-formats.html#PROTOCOL-MESSAGE-FORMATS-COMMANDCOMPLETE

Reply:
Yeah it's duplicated and shouldn't be, so removal makes sense.